components/SignUpForm/SignUpForm.tsx

```diff
@@ -42,19 +37,30 @@ export const SignUpForm = () => {
     resolver: yupResolver(validationSchema),
   });
 
+  const handleAuthErrors = (error: string) => {
```

Comment:
i suggest move switch case functiuon outside component (how do you think why?) and here left just setErrorAuth(getAuthError(error))